### PR TITLE
Notification

### PR DESCRIPTION
**Pull request by chinexman**
### What this PR does (required):
notification model is created, and it allow for a notification collection to be created in the database.
it describe the fields for title, description of the notification, whenever a notification is created, as well as the status and Date of the notification.

### Screenshots / Videos (front-end only):
- Post at least one screenshot to show the feature in action. **Only required in front-end work**

### Any information needed to test this feature (required):
- Post what steps to take to be able to test this feature (eg. "sign up as a new user and upload a profile picture from the navbar")

### Any issues with the current functionality (optional):
- List any problems where you were not able to complete all tasks on ticket
- List any problems this PR may cause for the project or other active PRs
- Post a screenshot/video of the problem, along with detailed console outputs where applicable

### CODE REVIEW

File: pull_request_template.md [hunk 1]
```diff
@@ -1,13 +1,18 @@
 ### What this PR does (required):
-- list PR functionality here
+
```
Comment (rajivtitus) [inline pull_request_template.md:2]: Please avoid editing the PR template, you can instead enter a description after you've successfully created a PR on Github :)

File: server/models/notification.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  },
+  title: {
+    type: String,
+    required: true,
+  },
+  description: {
+    type: String,
+    required: true,
+  },
+  readunread: {
```
Comment (rajivtitus) [inline server/models/notification.js:17]:
```suggestion
  read: {
```
Calling this 'readunread' seems a bit confusing

File: server/models/notification.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    type: String,
```
Comment (rajivtitus) [inline server/models/notification.js:18]:
```suggestion
    type: String,
    default: false,
```
You can add a default of 'false' as all notifications will be unread at first :)

File: server/models/notification.js [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+    required: true,
+  },
+  Date: {
+    type: Date,
```
Comment (rajivtitus) [inline server/models/notification.js:22]: We have a date type but we're using enum strings, could this be a mistake? 

Suggestion: You can get rid of the 'Date' field and use mongoose [timestamps](https://masteringjs.io/tutorials/mongoose/timestamps)

File: server/models/notification.js [hunk 5]
```diff
@@ -1,29 +1,28 @@
...
+    },
+    description: {
+      type: String,
+      required: true,
+    },
+    read: {
+      type: String,
+      required: true,
+      default: false,
+    },
```
Comment (ezeilo-su) [inline server/models/notification.js:22]: :+1:

**Review by ezeilo-su — COMMENTED** [review]
Nice update!

File: server/models/notification.js [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+  },
+  { timestamps: true }
```
Comment (rajivtitus) [inline server/models/notification.js:24]: 👍🏽

**Review by rajivtitus — APPROVED** [review]
LGTM! Good work with those changes 👍🏽

**Comment by rajivtitus** [comment]
@sholabioku @gagandeep95 Can I please get one of you to merge this in? 😄

**Pull request merged**